pub/index.html

```diff
@@ -0,0 +1,168 @@
+<!doctype html>
+<html>
+  <head>
+    <title>south park blog</title>
+    <style>
+    body {
+      margin: 0px;
+      padding: 0px;
+    }
+    header {
+      background-image: url(http://www.importantstudio.com/ironyard/spbusstop.jpg);
+      margin: 0;
+      width: 1280px;
+      height: 350px;
+      margin: 0px;
+    }
+    .container {
+        width: 850px;
+        padding: 10px;
+        margin: 0 auto 0 auto;
+        height: 1000px;
```

Comment:
Setting an explicit height can get you in trouble... As I'm sure you've noticed.